survey/locale/ja/LC_MESSAGES/django.po

```diff
@@ -245,19 +245,23 @@ msgstr "すべてのユーザーが閲覧・回答できる"
 msgid "Only authenticated users can see it and answer it"
 msgstr "認証されたユーザーのみ閲覧・回答できる"
 
-#: survey/models/survey.py:16
+#: survey/models/survey.py:17
+msgid "Users can edit their answers afterwards"
+msgstr "ユーザーは後で回答を編集できます"
```

Comment:
Do you know Japanese? :)

Reply:
No idea, it was just google translate. In fact, I had trouble with the translation tests which were failing and I thought it was because of the translation missing. I should maybe delete them.

Reply:
Let's keep it, I think it's better to have something, even if it's badly translated. (I think there is a test to check that the translation is "stable" ie no changes when launching make messages, but it does not force you to know 5 languages to fix :) )